pkg/api/v1beta1/dynakube/feature_flags.go

```diff
@@ -38,15 +38,10 @@ const (
 
```

Comment:
Question: Are we then completely removing the use cases of these feature flags? 
I'm asking because I'm working on the new Activegate CRD spec, and if so, I shouldn't include these.

Reply:
They should be removed completely.

Reply:
[Here](https://github.com/Dynatrace/dynatrace-operator/blob/010623f6bdc6601856fffa71b44944a7dd2ee925/pkg/webhook/validation/dynakube/deprecation.go#L27) you can see what should be moved